assets/src/styles/blocks/HubspotForm/HubspotFormEditorStyles.scss

```diff
@@ -0,0 +1,81 @@
+@import "../../master-theme/assets/src/scss/base/variables";
+@import "../../master-theme/assets/src/scss/base/colors";
+@import "../../master-theme/assets/src/scss/base/mixins";
+
+@import "HubspotFormSidebarStyles";
+
+.hubspot-form-editor {
+  padding-right: 32px;
+  padding-left: 32px;
+
+  & > .container {
```

Comment:
If you do end up removing the `container` class from the block, you won't need this anymore